.github/workflows/mambatest.yml

```diff
@@ -21,7 +21,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        #python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.12"]
```

Comment:
Has this to be reverted?

Reply:
not really, it is a lot of calculation, we might want to avoid